pica2mods-runner/src/main/resources/beans.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
XML bean configuration is a little bit "old school".
I suggest to load the data as nested properties:
https://stackoverflow.com/questions/33699171/need-help-figured-out-how-to-load-nested-list-from-yml-in-spring-boot-java-app
https://devwithus.com/spring-boot-configuration-properties/

This would be the configuration as properties (if we do not use yaml):
```
pica2mods.runner.catalogs.ubbs.url=https://opac.lbs-braunschweig.gbv.de/DB=1/
pica2mods.runner.catalogs.ubbs.srudbUrl=opac-de-84
pica2mods.runner.catalogs.ubbs.unapiUrl=opac-de-84
pica2mods.runner.catalogs.ubbs.unapiUrl=opac-de-84
pica2mods.runner.catalogs.ubbs.xsl=pica2mods.xsl
pica2mods.runner.catalogs.ubr...
``` 

And finally I suggest `sruKey` and `unapiKey` instead of `sruUrl` an `unapiUrl`

Reply:
I would prefer YAML...